smarts/core/sumo_road_network.py

```diff
@@ -522,7 +542,8 @@ def point_is_within_road(self, point):
         #
         # return False
 
-        nearest_lanes = self.nearest_lanes(point[:2], radius=5)
+        radius = max(5, 2 * self._default_lane_width)
```

Comment:
I think this with the default lane width at 3.2 would change some behaviour, I think these radius values were just arbitrary to begin with though.

Reply:
Yeah, that was my sense too (just from looking at the history).  I did recognize this as a change, but it didn't seem to affect any tests.  I think the reason it's ok is because this just means that the query to the Sumo net file might return a lane when it didn't use to, but we still check (and/r sort)  the lanes that are returned, so the extra-returned lane doesn't have any impact.